lib/fog/ovirt/compute/v4.rb

```diff
@@ -190,7 +190,7 @@ def blank_template
             @blank_template ||= client.system_service.get.special_objects.blank_template
           end
 
-          def create_search_by_datacenter(search:, datacenter:, page: nil)
+          def create_search_by_datacenter(search: nil, datacenter: nil, page: nil)
```

Comment:
Can you please add guard conditions to the function? You are relaxing requirements for the function, but it should not be possible to call it without these two parameters.

Reply:
any ideas on how to make it?

Reply:
Allright!

Updated in this PR

thanks